build/push-ci-build.sh

```diff
@@ -25,7 +25,7 @@ KUBE_GCS_NO_CACHING=n
 KUBE_GCS_MAKE_PUBLIC=y
 KUBE_GCS_UPLOAD_RELEASE=y
 KUBE_GCS_RELEASE_BUCKET=kubernetes-release
-KUBE_GCS_PROJECT=google-containers
+KUBE_GCS_PROJECT=google_containers
```

Comment:
Quick inspection reveals this isn't even used and should be deleted. The buckets are [globally namespaced](https://cloud.google.com/storage/docs/json_api/v1/buckets).